packages/cli/src/utils/runCommandSync.ts

```diff
@@ -58,10 +58,21 @@ export const runCommandSync = ({
   cwd?: string
 }) => {
   try {
-    execSync(debug ? `${cmd} --debug --verbose 2>&1` : `${cmd} 2>&1`, {
-      stdio: 'pipe',
-      cwd,
-    })
+    if (debug) {
+      console.log(`[STARTED] ${cmd}`)
+    }
+
+    const res = execSync(
+      debug ? `${cmd} --debug --verbose 2>&1` : `${cmd} 2>&1`,
+      {
+        stdio: 'pipe',
+        cwd,
+      }
+    )
+    if (debug) {
+      console.log(`[STATUS] ${res.toString()}`)
```

Comment:
Show logs related to the commands previously hidden. (e.g. generate:schema that now you can see when the scheme is generated successfully).